library/swift/test/BUILD

```diff
@@ -77,3 +77,10 @@ envoy_mobile_swift_test(
         "CounterTests.swift",
     ],
 )
+
+envoy_mobile_swift_test(
+    name = "element_tests",
+    srcs = [
+        "ElementTests.swift",
+    ],
+)
```

Comment:
Trailing space 😃

Reply:
it wasn't here previously so I though that it was not needed 🤔

Reply:
<img width="94" alt="Screen Shot 2020-09-18 at 3 42 46 PM" src="https://user-images.githubusercontent.com/2643476/93651232-9ad8e280-f9c5-11ea-83a5-0ad0c3f29b5a.png">

I was referring to this ^ (not showing up on the main GitHub screen for some reason, only when you view files)